metadeploy/labels_fr_CA.json

```diff
@@ -0,0 +1,102 @@
+{
```

Comment:
@dmkud - this file may be an "extra", duplicating `labels_fr-CA.json`, but I don't want to remove it from this PR if there might be some updates we bring into that other file ... any thoughts on how to approach that?